inst/tutorials/chapter3/chapter3.Rmd

```diff
@@ -276,8 +428,10 @@ fluidRow(
 output$distPlot3 <- renderPlot({
   set.seed(2)
   nsample = rnorm(n=input$ssize,mean=5,sd=2)
-  hist(nsample,col = 'darkgray', border = 'white',freq=FALSE,main=paste0("Histogram with N = ",input$ssize))
+  hist(nsample,col = 'darkgray', border = 'white',freq=FALSE,main=paste0("Histogram with N = ",input$ssize), xlim = c(0, 10))
   lines(density(nsample),col="red",lw=2)
+  lines(seq(0, 10, .01), dnorm(x = seq(0, 10, .01), mean = 5, sd = 2), col = 'green')
+  legend("bottom",c("Normal Distribution","Sampling Distribution"),col=c("green","red"),lty=c(1,1),lw=c(2,2), cex = 0.75) #Ideally, put legend in the leftmost column of the Shiny app (under the slider), but needs HTML + CSS --> feasible but probably not worth it...
```

Comment:
yes, dont bother with that. let's leave it like this, its good. We need some explanation of both red and green lines just before the plot.